Source/ACE.Server/Managers/PropertyManager.cs

```diff
@@ -469,8 +475,9 @@ public static void LoadDefaultProperties()
                 ("char_delete_time", 3600),         // the amount of time in seconds a deleted character can be restored
                 ("mansion_min_rank", 6),            // overrides the default allegiance rank required to own a mansion
                 ("max_chars_per_account", 11),      // retail defaults to 11, client supports up to 20
-                ("pk_timer", 20)                    // the number of seconds where a player cannot perform certain actions (ie. teleporting)
+                ("pk_timer", 20),                   // the number of seconds where a player cannot perform certain actions (ie. teleporting)
                                                     // after becoming involved in a PK battle
+                ("player_save_interval", 5)         // number of minutes between saving of characters
```

Comment:
Seconds make sense here. It's the unit of measurement we use in the majority (all?) of cases in ACE

Reply:
I was just using the existing units which was minutes. Seconds works too. I'll switch it over.